eq-author/src/components/Header/SharingModal/index.js

```diff
@@ -75,18 +78,36 @@ const ActionButtonGroup = styled(ButtonGroup)`
   position: relative;
 `;
 
+const confirmEditorRemoval = (removeEditor, currentUser, user, history) => {
+  if (currentUser.id === user.id) {
+    const confirmed = window.confirm(
```

Comment:
Should we be using the window.confirm rather than a modal? Check with @hamishtaplin

Reply:
window.confirm is okay with Hamish